### PR TITLE
ARCH-1049 - Updates to cypress template after migration

### PR DESCRIPTION
**Pull request by danielle-casella-adams**

### CODE REVIEW

**Review by JosephDSchwartz — APPROVED** [review]

File: workflow-templates/im-test-cypress.yml [hunk 1]
```diff
@@ -59,7 +75,7 @@ jobs:
           dotnet-version: ${{ env.DOTNET_VERSION }}
 
       - name: dotnet build
-        working-directory: ./ # TODO: verify the root is the correct location for the solution file
+        working-directory: . # TODO: verify the root is the correct location of the solution file
```
Comment (hpractv) [inline workflow-templates/im-test-cypress.yml:78]: Will this plain dot get us mapped back to the git root?

File: workflow-templates/im-test-cypress.yml [hunk 2]
```diff
@@ -82,12 +98,12 @@ jobs:
...
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }} # Special per-job token generated by GH for interacting with the repo
-          results-file: '${{ env.JS_APP_DIR }}/${{ env.CYPRESS_RESULTS_FILE }}'
+          results-file: '${{ env.JS_TEST_DIR }}/${{ env.CYPRESS_RESULTS_FILE }}'
           report-name: 'Cypress ${{ github.run_number }}'
           # TODO: Verify whether you want status checks or pr comments with the results.
           create-status-check: true
-          create-pr-comment: false
-          # update-comment-if-one-exists: false # TODO: By default this creates one comment and updates it for each run.  Uncomment if you wish to have one new comment for every workflow run.
+          create-pr-comment: true
```
Comment (hpractv) [inline workflow-templates/im-test-cypress.yml:105]: 👍🏽 This does make more sense for a starting default.

File: workflow-templates/im-test-cypress.yml (hunk 1 above)
Comment (danielle-casella-adams) [reply workflow-templates/im-test-cypress.yml:78]: yeah and the auto-plan-and-cmment-on-pr workflow sets it that way.  I'm not sure why I had the ./ to begin with here.

**Review by hpractv — APPROVED** [review]
👍🏽

**Pull request merged**